WcaOnRails/app/models/competition.rb

```diff
@@ -1393,8 +1393,9 @@ def set_wcif_events!(wcif_events, current_user)
 
     # Update all events.
     wcif_events.each do |wcif_event|
-      event_to_be_added = wcif_event["rounds"]
-      if event_to_be_added
+      event_to_be_updated = wcif_event["rounds"]
+      if event_to_be_updated
+        raise WcaExceptions::BadApiParameter.new("Cannot update events on a competition with results posted") unless current_user.can_update_events?(self)
```

Comment:
The line is so long that it would be better to just wrap that in an `unless` block instead. Also the message is quite specific as compared to the method name, so perhaps something like `"Cannot update events."`.

Reply:
@jonatanklosko This line's length is the same as [#L1379](https://github.com/thewca/worldcubeassociation.org/blob/dba2f048f71f818ce35be10d708292e45c042049/WcaOnRails/app/models/competition.rbL1379) and the reason is similar. If this should be changed, do I need to change [L1379](https://github.com/thewca/worldcubeassociation.org/blob/dba2f048f71f818ce35be10d708292e45c042049/WcaOnRails/app/models/competition.rb#L1379) and [L1389](https://github.com/thewca/worldcubeassociation.org/blob/dba2f048f71f818ce35be10d708292e45c042049/WcaOnRails/app/models/competition.rb#L1389)?

Reply:
I believe that would make sense, yeah!

Reply:
@jonatanklosko I've made the changes. Please have a look.

Reply:
LGTM =)